Gemfile.rails_version

```diff
@@ -2,13 +2,15 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV["RAILS_VERSION"] || '4.2.10'
+env_rails_version = ENV['RAILS_VERSION'] || '5.1.7'
 
 gem "rails", "~> #{env_rails_version}"
 
 case env_rails_version
 when "master"
   {github: "rails/rails"}
+when /^5.1/
+  gem 'bundler', '~> 2.1'
```

Comment:
this may be redundant since i added it as a gem dependency in the gemspec

Reply:
Yup -- I'd say let's remove it so we can remove this whole split. Do we need the other dependencies from below in 5.1 (i.e. should we change the line below to be `when /^5.1/, /^5.0/, /^4.2.10/`? Also, can we remove `simple_form` from the other sections here since it's no longer necessary.

Reply:
Updated